powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/ActivationHistoryServiceBehavior.java

```diff
@@ -143,7 +143,7 @@ public ActivationHistoryResponse getActivationHistory(String activationId, Date
     private void logAuditItem(ActivationRecordEntity activation, String externalUserId, String historyEventReason) {
         // Prepare shared parameters
         final AuditDetail.Builder auditDetailBuilder = AuditDetail.builder()
-                .type("activation")
+                .type(AuditingServiceBehavior.AuditType.ACTIVATION.getCode())
```

Comment:
I took inspiration from enrolment-server-wultra project. Specifically AuditService class. Frankly, I am not sure whether It would be also better to create audit function in AuditingServiceBehavior class for each AuditType, similarly like in AuditService class from enrolment-server like this:
` public void audit(final IdentityVerificationEntity identityVerification, final String message, final Object... args) {
        final AuditDetail auditDetail = createAuditDetail(AuditType.IDENTITY_VERIFICATION, identityVerification);
        audit.info(message, auditDetail, args);
    }`